Controller/MediaAdminController.php

```diff
@@ -116,16 +116,12 @@ private function setFormTheme(FormView $formView, $theme)
     {
         $twig = $this->get('twig');
 
-        try {
-            $twig
-                ->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')
-                ->setTheme($formView, $theme);
-        } catch (\Twig_Error_Runtime $e) {
-            // BC for Symfony < 3.2 where this runtime not exists
-            $twig
-                ->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
-                ->renderer
-                ->setTheme($formView, $theme);
+        // BC for Symfony < 3.2 where this runtime not exists
+        if (method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
+            $twig->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')->setTheme($formView, $theme);
+        } else {
+            $twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
+                ->renderer->setTheme($formView, $theme);
```

Comment:
Using an early return statement, you can make the code even more similar to what it will be when we drop sf <3.2 : 

```php
if (!method_exists('Symfony\Bridge\Twig\AppVariable', 'getToken')) {
    $twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
        ->renderer->setTheme($formView, $theme);
        return;
}
$twig->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')->setTheme($formView, $theme);
```